bsd/models.py

```diff
@@ -64,7 +65,12 @@ def find_constituents_by_state_cd(state_cd):
     filter = {}
     filter['state_cd'] = str(state_cd)
     filter['is_subscribed'] = True
-    bundles = ['primary_cons_addr', 'primary_cons_email']
+
+    """Check if member of sendable cons group"""
+    if EVENTS_PROMOTE_SENDABLE_CONS_GROUP_ID:
+        filter['cons_group'] = EVENTS_PROMOTE_SENDABLE_CONS_GROUP_ID
+
+    bundles = ['primary_cons_addr', 'primary_cons_email','cons_group']
```

Comment:
why do we need cons group bundle? remove?

Reply:
Done.